docs/running-on-kubernetes.md

```diff
@@ -926,6 +926,13 @@ specific to Spark on Kubernetes.
    <code>spark.kubernetes.executor.podTemplateFile=/path/to/executor-pod-template.yaml`</code>
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.deleteExecutors</code></td>
```

Comment:
Should this be named `spark.kubernetes.executor.deleteOnTermination` so the style is consistent with other executor-related options?

Reply:
No problem I can add change to that seems more readable if we remove driver pods as well.